src/pages/ResultsListPage.js

```diff
@@ -101,8 +101,11 @@ class ResultsListPage extends BaseResultsPage {
     }
 
     renderPersonalisationLink(): ReactElement {
+        const current = this.props.location.pathname;
+        const separator = current.endsWith("/") ? "" : "/";
```

Comment:
The path isn't an array - we could split it on '/' and remove all the empty strings from the resulting array, but that also sounds like hacks.